content/source/docs/cloud/integrations/service-now/service-catalog.html.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: "cloud"
+page_title: "Service Catalog - ServiceNow Service Catalog Integration - Terraform Cloud and Terraform Enterprise"
+description: |-
+  Configure Service Catalogs and VCS repositories to allow end users to
+  provision infrastructure using ServiceNow and Terraform Cloud.
+---
+
+# Service Catalog Configuration
+
+When using ServiceNow with the Terraform Cloud integration, you will configure
+at least one Service Catalog. You will also configure one or more version
+control system (VCS) repositories containing the Terraform configurations which
+will be used to provision that infrastructure. End users will request
+infrastructure from the service catalog, and Terraform Cloud will fulfill the
+request by creating a new workspace, applying the configuration, and then
+reporting the results back to ServiceNow.
+
+## Prerequisites
+
+1. The Terraform Cloud integration software installed and configured on
+   your ServiceNow instance. These steps are covered in the [installation
+   documentation](./index.html).
+1. The OAuth token ID and repository identifier for each VCS repository that
+   Terraform Cloud will use to provision infrastructure. Your Terraform Admin
+   will provide these to you.
+1. Any Terraform or environment variables required by the configurations in the
+   given VCS repositories.
```

Comment:
Yeah, they're just different kinds of items. How about smashing it apart, like this? 

```suggestion
Before continuing in this document, make sure that the Terraform Cloud integration software
is installed and configured on your ServiceNow instance. These steps are covered in the [installation
documentation](./index.html).

Additionally, you must have have the following information: 

1. The OAuth token ID and repository identifier for each VCS repository that
   Terraform Cloud will use to provision infrastructure. Your Terraform Admin
   will provide these to you.
1. Any Terraform or environment variables required by the configurations in the
   given VCS repositories.
```